src/Microsoft.Identity.Client/Internal/Interfaces/IWebUIFactory.cs

```diff
@@ -29,6 +29,6 @@ namespace Microsoft.Identity.Client.Internal.Interfaces
 {
     internal interface IWebUIFactory
     {
-        IWebUI CreateAuthenticationDialog(IPlatformParameters parameters, UIBehavior behavior, RequestContext requestContext);
+        IWebUI CreateAuthenticationDialog(UIParent parent, RequestContext requestContext);
```

Comment:
AuthenticationDialog is misleading here. i.e. in the case of android, you need activity but dialog means something different.